src/components/splash/Splash.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react'
-import { StyleSheet, View } from 'react-native'
+import { ImageBackground, StyleSheet } from 'react-native'
 import AnimationsLogo from '../common/animations/Logo'
-import wavePattern from '../../assets/wave50.svg'
+import wavePattern from '../../assets/splashWaves.svg'
 import Wrapper from '../common/layout/Wrapper'
 import Section from '../common/layout/Section'
 import Config from '../../config/config'
 
 const Splash = ({ animation }) => (
   <Wrapper style={styles.wrapper}>
     <Section style={styles.container}>
-      <View style={styles.backgroundWaves} />
-      <Section.Stack style={styles.content} grow justifyContent="center">
-        <AnimationsLogo animation={animation} />
-        <Section.Text fontSize={22} color="darkBlue">
-          {`V${Config.version}`}
-        </Section.Text>
-      </Section.Stack>
+      <ImageBackground
+        source={wavePattern}
+        imageStyle={styles.waves}
+        style={styles.backgroundWaves}
+        resizeMode="repeat"
+      >
+        <Section.Stack style={styles.content} grow justifyContent="center">
+          <AnimationsLogo animation={animation} />
+          <Section.Text fontSize={22} color="darkBlue">
+            {`V${Config.version}`}
```

Comment:
why not just 
```
V{Config.version}
```
?

Reply:
does that work?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I mean that we don't need template literals here.
We could just print "V" letter then version value as jsx expression:

```
<Section.Text fontSize={22} color="darkBlue">
   {`V${Config.version}`} // jsx expression including literal containing 
                                          // embedded expression with version value
</Section.Text>

// the same as above but looks easier
<Section.Text fontSize={22} color="darkBlue">
   V{Config.version} // just an  jsx expression  with version value
</Section.Text>
```

Reply:
@serdiukov-o-nordwhale 
I don't know why) there are many places where the text passed like this. Maybe to prevent some illegal code in var or something else.
But if you want i can change it.

@sirpy what do you think?

Reply:
@yaroslav-fedyshyn-nordwhale 
Any jsx expression (doesnt matter is is a direct varible reference or an literal containing embedded JS expression) anyway cleans for illegal code (e.g. html tags. js etc) before passing to the virtual DOM

The only way to output unescaped code is:
https://ru.reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml

Reply:
@serdiukov-o-nordwhale you right